sample/src/main/java/com/canhub/cropper/sample/camera/app/SCameraFragment.kt

```diff
@@ -56,6 +53,15 @@ internal class SCameraFragment :
         ActivityResultContracts.RequestPermission()
     ) { isGranted: Boolean -> presenter.onPermissionResult(isGranted) }
 
+    private val pickImage =
+        registerForActivityResult(PickImageContract(), presenter::onPickImageResult)
```

Comment:
Please, let's avoid to use the reflection `::`. I agree is much better in an aesthetics perspective, I tend to avoid reflection in general + some times Android Studio go crazy not finding it usages making harder for debug.

`registerForActivityResult(PickImageContract()) { presenter.onPickImageResult(it) }`

Reply:
:raised_eyebrow: